File: client/src/app/models/product.ts
export interface Product {
  id: number;
  name: string;
  information1: string;
  information2: string;
  information3: string;
  information4: string;
  price: number;
  pictureUrl: string;
  type: string;
  quantityInStock: number;
}

export interface ProductParams {
  orderBy: string;
  searchTerm?: string;
  types: string[];
  pageNumber: number;
  pageSize: number;
}

File: API/Entities/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Information1 { get; set; }
        public string Information2 { get; set; }
        public string Information3 { get; set; }
        public string Information4 { get; set; }
        public long Price { get; set; } // Stripe uses this,for future implementation,also SQLite doesn't recognize decimals
        public string PictureUrl { get; set; }
        public string Type { get; set; }
        public int QuantityInStock { get; set; }
    }
}